Sources/NetworkProtection/Notifications/NetworkProtectionNotification.swift

```diff
@@ -21,6 +21,18 @@ import Common
 
 #if os(macOS)
 
+fileprivate extension Bundle {
+    private static let networkProtectionDistributedNotificationPrefixKey = "DISTRIBUTED_NOTIFICATIONS_PREFIX"
+
+    var networkProtectionDistributedNotificationPrefix: String {
+        guard let bundleID = object(forInfoDictionaryKey: Self.networkProtectionDistributedNotificationPrefixKey) as? String else {
+            fatalError("Info.plist is missing \(Self.networkProtectionDistributedNotificationPrefixKey)")
+        }
+
+        return bundleID
+    }
+}
+
```

Comment:
This was moved to the top to highlight it a bit more, since it's an extension that's used by the code in this file.  Nothing was changed from the original code though.